.github/workflows/post-merge-jobs.yml

```diff
@@ -98,6 +100,18 @@ jobs:
               with:
                   name: build
                   path: build
+            - name: Login to Azure
```

Comment:
Ideally more of this would be shared between Jobs and the APIs but Jobs doesn't use the shared action because its deployment step is different.  I might look into using a shared action in the future.